src/core/a-entity.js

```diff
@@ -266,12 +264,17 @@ var proto = Object.create(ANode.prototype, {
           // For scene default components, expose them in the DOM.
           HTMLElement.prototype.setAttribute.call(this, name, this.defaultComponents[name]);
         }
+
+        // Check if component already defined.
+        if (name in this.components) { return; }
```

Comment:
I changed some comments.

`isComponentDefined` checks if the component *should be defined* for the entity. What we want to check here if the component *has already been initialized* for the entity.